packages/tenderly-actions/src/actions/networkConfig.ts

```diff
@@ -0,0 +1,421 @@
+import { Network } from "@tenderly/actions";
+import { BigNumber, ethers } from "ethers";
+
+export type NetworkConfig = {
+  [key in Network]:
+    | {
+        chainId: number;
+        isTestnet?: boolean;
+        isSupported: true;
+        riskStewardReceiver: {
+          address: `0x${string}`;
+          gasLimit: number;
+          gasPrice: BigNumber;
+        };
+      }
+    | {
+        isSupported: false;
+        isTestnet?: boolean;
+      };
+};
```

Comment:
```suggestion
  [key in SupportedNetwork]:
    {
        chainId: number;
        isTestnet?: boolean;
        riskStewardReceiver: {
          address: `0x${string}`;
          gasLimit: number;
          gasPrice: BigNumber;
        };
      }
};
```

To simplify this, you could extract a `SupportedNetwork` type from `Network`, instead of all the keys.

Then you can remove the `isSupported` flag and...